scripts/parse.py

```diff
@@ -73,8 +77,11 @@ def offset2string(offset):
     ''' Put a ` character after the offset to facilitate text searches '''
     return '%08X`  ' % offset
 
-def desc2suffix(desc):
-    return ' (%s)' % desc if desc else ''
+def desc2suffix(desc: str, valuedescSet: Optional[Set[str]] = None) -> str:
```

Comment:
This function should only be responsible for turning a description string into something that can be appended to the already created output. It should not have any logic related to valuedescset & co.

Reply:
valuedescSet is used to append an extra suffix if we know anything about the value. Could easily be split into a different fucntion, it's just it would always be called alongside desc2suffix so there'd be a bit of duplication.